### PR TITLE
Changes to overall layouts and fixes certain bugs/annoyances. Subjective update.

### LINKED ISSUE #17: Text overflow onto update images
CleanShot 2024-08-15 at 05 38 50@2x
CleanShot 2024-08-15 at 05 38 41@2x
CleanShot 2024-08-15 at 05 38 35@2x

Thrown some examples here, cool project, I'll try and fix it.

### PR DESCRIPTION
**Pull request by insomnia-creator**
Fixes #17(by differentiating content container and sidebarImage container)

Also left-aligns content and spans the header across the page.

Additionally changes the navbar and minimises the search box on desktop. Haven't changed much on mobile, mainly a desktop change.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -224,20 +226,29 @@ header {
   background-color: var(--bg-color);
   color: var(--text-color);
   transition: border-color 0.3s, box-shadow 0.3s;
-  height: 40px; /* Set a fixed height to match the select */
+  /* height: 40px; /* Set a fixed height to match the select */
```
Comment (insomnia-creator) [inline styles.css:229]: This is a bug which seems to break on firefox + webkit?

Comment (insomnia-creator) [reply styles.css:229]: CleanShot 2024-08-15 at 08 06 22@2x
Firefox
CleanShot 2024-08-15 at 08 06 16@2x
Webkit based Safari(sadly the styling on webkit is a PITA so not changing much there)

Comment (Jalenzzz) [reply styles.css:229]: I main floorp, a fork of firefox and on my end it seems fine. 
So for you on firefox, not even the news ticker shows? Or is the screenshot cropped
image

**Review by Jalenzzz — CHANGES_REQUESTED** [review]
Good stuff except the search results are on the far left from the search container.
So I guess that can be changed so that the results appear below the search container.
Before:
image

After:
image

File: styles.css (hunk 1 above)
Comment (insomnia-creator) [reply styles.css:229]: No I meant to use firefox as an example of ideal structure, poor phrasing on my end.

**Comment by insomnia-creator** [comment]
Hmm... is this a browser specific issue? Can't seem to point my finger to it.

https://github.com/user-attachments/assets/efa04f4b-8a2d-4cbb-8f23-41a09d16650a

File: styles.css (hunk 1 above)
Comment (insomnia-creator) [reply styles.css:229]: Didn't answer your second question, the screenshot is cropped.

**Comment by insomnia-creator** [comment]
Checked it on both Safari + Chromium too... it would be a great help if you could please check the styling on your browser.

**Comment by Jalenzzz** [comment]
I've seen the issue, I forgot to change the display style of the container to flex from block according to your changes lolll. 🤦🏾 It's working well now!
[Screencast from 15-08-24 20:04:42.webm](https://github.com/user-attachments/assets/6e89a7d3-ff42-473b-80fd-7d5c4ef82c5a)

**Review by Jalenzzz — APPROVED** [review]

**Pull request merged**